File: C#/Урок 038 - while/Задача 06.cs
class Program
{
    static void Main()
    {
        int computerNumber = 27;
        int userNumber;
        int triesCount = 1;

        System.Console.Write("Try to guess computer number (from 0 to 100): ");
        userNumber = System.Convert.ToInt32(System.Console.ReadLine());
        while (userNumber != computerNumber)
        {
            System.Console.Write("You're wrong. Try again: ");
            userNumber = System.Convert.ToInt32(System.Console.ReadLine());
            triesCount++;
        }

        System.Console.WriteLine($"You win! Number of tries: {triesCount}");
    }
}

File: C++/Урок 032 - while/Задача 06.cpp
#include <iostream>

int main()
{
    int computerNumber = 27;

    std::cout << "Guess computer's number. It guessed a number from 0 to 100: ";
    int userNumber;
    std::cin >> userNumber;

    int triesAmount = 1;

    while (userNumber != computerNumber)
    {
        std::cout << "You didn't guess. Try again: ";
        triesAmount++;
        std::cin >> userNumber;
    }

    std::cout << "You guessed! My congratulations! You used " << triesAmount << " tries.\n";

	return 0;
}